flexmeasures/data/models/reporting/tests/test_tibber_reporter.py

```diff
@@ -172,36 +169,45 @@ def tibber_test_data(fresh_db, app):
 
     electricity_price = GenericAsset(name="Electricity Price", generic_asset_type=price)
 
-    VAT = GenericAsset(name="VAT", generic_asset_type=tax)
+    VAT_asset = GenericAsset(name="VAT", generic_asset_type=tax)
 
     electricity_tax = GenericAsset(name="Energy Tax", generic_asset_type=tax)
 
-    db.session.add_all([electricity_price, VAT, electricity_tax])
+    db.session.add_all([electricity_price, VAT_asset, electricity_tax])
 
     # Taxes
-    BWV = Sensor("BWV", generic_asset=VAT, event_resolution=timedelta(days=365))
-    EB = Sensor(
-        "EB", generic_asset=electricity_tax, event_resolution=timedelta(days=365)
+    VAT = Sensor(
+        "VAT",
+        generic_asset=VAT_asset,
+        event_resolution=timedelta(days=365),
+        unit="unit range",
```

Comment:
Our unit conversion utilities only support conversion of units it recognizes, using the `pint` package. I suspect "unit range" is not one of them. Instead, use `unit=""` to define unitless sensor data.

Btw, VAT is not by definition restricted to the unit range. and could in principle be negative or exceed 100%.

Reply:
The VAT percentage is defined with respect to the price excl. tax, not with respect to the price incl. tax. (Maybe you're thinking of income tax?)

Afaik there's no theoretical limit to what the VAT rate could be. In principal a government could decide to levy VAT at a rate of 105%. An item priced at 100 euros excl. tax would then be priced at 205 euros incl. tax.